src/link/style.scss

```diff
@@ -360,6 +360,45 @@
 				}
 
 			}
+
+			@media screen and (max-width: 782px) {
+				width: 120px;
+				height: 85px;
+				background-color: #fff;
+				color: #000;
+				border-top-right-radius: unset;
+				border-bottom-right-radius: unset;
+				border: solid 1px #ccc;
+				font-size: 16px;
+				box-shadow: 0 2px 2px #1e1e1e;
+
+				&-menu {
+					height: 20px;
+					width: 20px;
+					background-image: url(./images/vertical-ellipsis.svg);
+					margin: auto 10px;
+				}
+
+				&-change {
+					text-align: center;
+
+					&::before {
+						content: url(./images/change-black.svg);
+						left: -7px;
+						top: 5px;
+					}
+				}
+
+				&-remove {
+					text-align: center;
+
+					&::before {
+						content: url(./images/remove-black.svg);
+						left: -7px;
+						top: 5px;
+					}
+				}
+			}
```

Comment:
the RTL preview UI looks like this, you can find the `dir` attribution in class `wikipediapreview`
<img width="533" alt="image" src="https://github.com/wikimedia/wikipediapreview-wordpress/assets/2560096/abf5164a-b944-43a7-a141-8532dd01ba81">